addons/ui/flexiMenu/fnc_add.sqf

```diff
@@ -11,9 +11,9 @@ _msg = "FLEXIMENU: Unknown Error in fnc_add.sqf"; //Initialize
 _key = [];
 
 // validate params
-_msg = format ["Error: invalid params. %1 (%2)", _this, __FILE__ ];  //
+_msg = format ["Error: invalid params. %1 (%2)", _this, __FILE__];
```

Comment:
Reminder for myself to double check this.

Reply:
Well?